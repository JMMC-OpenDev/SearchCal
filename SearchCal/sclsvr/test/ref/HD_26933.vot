version https://git-lfs.github.com/spec/v1
oid sha256:55ee4fc4493d456d8a85e52cce46af11b0a3906e75670ca5a2ba3d8b0b3623a1
size 537421
