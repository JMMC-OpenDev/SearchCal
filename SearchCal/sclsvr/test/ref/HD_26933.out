version https://git-lfs.github.com/spec/v1
oid sha256:c35073157c1a4d46054b8a7ff8d742bdb519969edb8ad8cdeeb3a39f73bcefb0
size 678306
